cypress/integration/login.spec.ts

```diff
@@ -0,0 +1,24 @@
+const loginScreenTestIds = {
+  loginHeader: 'login-header',
+  loginEmail: 'login-form__input-email',
+  loginPassWord: 'login-form__input-password',
+  loginSubmit: 'login-form__button-submit',
+};
+describe('Login page', () => {
+  it('renders the login page with the right element', () => {
+    cy.visit('/');
+    cy.get('.app-header-title').should('have.text', 'Sign in to Nimble');
+    cy.findByTestId(loginScreenTestIds.loginEmail).should('exist');
+    cy.findByTestId(loginScreenTestIds.loginPassWord).should('exist');
+  });
+
+  describe('given valid credentials', () => {
+    it('redirects to survey homepage', () => {
+      cy.visit('/');
+      cy.findByTestId(loginScreenTestIds.loginEmail).type('suphanat@nimblehq.co');
+      cy.findByTestId(loginScreenTestIds.loginPassWord).type('12345678');
+      cy.findByTestId(loginScreenTestIds.loginSubmit).click();
```

Comment:
Just a small question, is this making a real API call? 

Normally we [intercept](https://docs.cypress.io/api/commands/intercept) the call and provide a fixture so the test doesn't fail to external factors

Reply:
Yes, it does I will implement that and let you know 😀

Reply:
@suphanatjarukulgowit Are you going to add that feature in this PR?